tartiflette/language/parsers/lark/graphql_sdl_grammar.lark

```diff
@@ -152,7 +152,7 @@ directive: "@" name arguments?
 directives: directive+
 directive_location: TYPE_SYSTEM_DIRECTIVE_LOCATION
                   | EXECUTABLE_DIRECTIVE_LOCATION
-directive_locations: directive_location ("|" directive_location)*
+directive_locations: "|"? directive_location ("|" directive_location)*
```

Comment:
Hum, what is this fixing ?

Reply:
This is to comply with the [GraphQL specification](http://spec.graphql.org/June2018/#DirectiveLocations) regarding directive locations.

Before this fix, we were not able to do this:
```graphql
directive @debug(message: DefaultRawString) on
    | FIELD_DEFINITION
    | ARGUMENT_DEFINITION
    | INPUT_FIELD_DEFINITION
    | ENUM
    | ENUM_VALUE
    | SCALAR
    | INPUT_OBJECT
```

We were forced to remove the `|` for this first location which was the reason that some of our tests was failling (those that I un-skipped (cf. `tests/unit/language/parsers/lark/test_parser.py`).

Reply:
Nice !